### ISSUE TITLE
Create hierarchical DELFI attribute data structure

### ISSUE THREAD

**Issue by Robbendebiene** [OPENER-next/OPENER-next#11]
The goal of this data structure is to easily allow the addition/removal of new attributes and its corresponding information. The UI should be generated from this dataset.

Every attribute should contain things like:
- DELFI id
- Name and description/question.
- The corresponding OSM tag
- Allowed values and value type
- Values contain optionally a list of child attributes that will be shown if the particular value is selected (attributes are referenced via DEFLI id)
- The order of the values describes the usage of the value (high usage means low index)

In question:
- Always provide a value selection but also always allow custom values?

Example JSON data structure

```
{
  id: 2,
  name: "Bahnsteig Typ",
  details: "",
  tag: "amenity",
  values: [
   {
     value: "1",
     name: "item 1",
     image: "nameoftheimage.svg",
     details: "some further details that describe this attribute",
     children: [3, 5]
   },
   {
     value: "2",
     name: "item 2",
     image: "nameoftheimage2.svg",
     details: "some further details that describe this attribute"
   },
   ...
  ]
}
```

**Comment by Robbendebiene** [comment]
For consistency, inherit documentation and error prevention/validation ideally a JSON schema (http://json-schema.org/) should be worked out here.

**Comment by 7h30n3** [comment]
Solved by #26
